eras/shelley-ma/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -793,12 +822,9 @@ prune assets =
   Map.filter (not . null) $ Map.filter (/= 0) <$> assets
 
 -- | Rather than using prune to remove 0 assets, when can avoid adding them in the
---   first place by using valueFromList to construct a MaryValue.
-valueFromList :: Integer -> [(PolicyID era, AssetName, Integer)] -> MaryValue era
-valueFromList ada =
-  foldr
-    (\(p, n, i) ans -> insert (+) p n i ans)
-    (MaryValue ada Map.empty)
```

Comment:
I think there is value keeping this function, re-defined in terms of multiAssetFromList, because it is used so many places.